crates/re_space_view/src/data_query_blueprint.rs

```diff
@@ -689,123 +522,117 @@ mod tests {
                 outputs: vec![
-                    "/",
-                    "/parent/",
+                    "/**",
+                    "/parent/**",
                     "/parent",
-                    "/parent/skipped/",
+                    "/parent/skipped/**",
                     "/parent/skipped",        // Included because an exact match
                     "/parent/skipped/child1", // Included because an exact match
                     "/parent/skipped/child2",
                 ],
             },
             Scenario {
-                inclusions: vec!["parent/skipped", "parent/skipped/child2", "parent/"],
-                exclusions: vec!["parent"],
+                filter: r"+ parent/skipped
+                          + parent/skipped/child2
+                          + parent/**
+                          - parent",
                 outputs: vec![
-                    "/",
-                    "/parent/", // Parent leaf has been excluded
-                    "/parent/skipped/",
+                    "/**",
+                    "/parent/**", // Parent leaf has been excluded
+                    "/parent/skipped/**",
                     "/parent/skipped",        // Included because an exact match
                     "/parent/skipped/child1", // Included because an exact match
                     "/parent/skipped/child2",
                 ],
             },
             Scenario {
-                inclusions: vec!["parent/"],
-                exclusions: vec!["parent/skipped/"],
-                outputs: vec!["/", "/parent"], // None of the children are hit since excluded
+                filter: r"+ parent/**
+                          - parent/skipped/**",
+                outputs: vec!["/**", "/parent"], // None of the children are hit since excluded
             },
             Scenario {
-                inclusions: vec!["parent/", "parent/skipped/child2"],
-                exclusions: vec!["parent/skipped/child1"],
+                filter: r"+ parent/**
+                          + parent/skipped/child2
+                          - parent/skipped/child1",
                 outputs: vec![
-                    "/",
-                    "/parent/",
+                    "/**",
+                    "/parent/**",
                     "/parent",
-                    "/parent/skipped/",
+                    "/parent/skipped/**",
                     "/parent/skipped/child2", // No child1 since skipped.
                 ],
             },
             Scenario {
-                inclusions: vec!["not/found"],
-                exclusions: vec![],
+                filter: r"+ not/found",
                 // TODO(jleibs): Making this work requires merging the EntityTree walk with a minimal-coverage ExactMatchTree walk
                 // not crucial for now until we expose a free-form UI for entering paths.
-                // vec!["/", "not/", "not/found"]),
+                // vec!["/**", "not/**", "not/found"]),
```

Comment:
I guess this is desired for setting up a blueprint for data that hasn't arrived yet?

Reply:
Exactly -- see above.